packages/ui/.eslintrc.js

```diff
@@ -17,10 +17,17 @@ module.exports = {
     tsconfigRootDir: __dirname,
   },
   rules: {
+    'import/no-extraneous-dependencies': [
+      'error',
+      { packageDir: ['.', '../..'] },
+    ],
     'no-console': 'error',
     'no-debugger': 'error',
+    'no-undef': 'off',
```

Comment:
Disabled `no-undef` because it does play well with/understand TS generics. Will remove once the standard linting config is used in the _ui_ package